### PR TITLE
feat: update session variable if set global variables

### LINKED ISSUE #1747: `session variables` have not update if `global variables` is set
df8e44ef1673f57694e218a467e49c51

### PR DESCRIPTION
**Pull request by keyu813**
close #1747 
init session variables from global variables when sr init

### CODE REVIEW

File: src/sdk/sql_cluster_router.cc [hunk 1]
```diff
@@ -2496,6 +2503,7 @@ ::hybridse::sdk::Status SQLClusterRouter::SetVariable(hybridse::node::SetPlanNod
         if (!cluster_sdk_->TriggerNotify(::openmldb::type::NotifyType::kGlobalVar)) {
             return {::hybridse::common::StatusCode::kRunError, "zk globlvar node not update"};
         }
+        session_variables_[key] = value;
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc]: remove return

File: src/sdk/sql_cluster_router.cc [hunk 2]
```diff
@@ -258,11 +258,18 @@ bool SQLClusterRouter::Init() {
...
-    // todo: init session variables from systemtable
-    session_variables_.emplace("execute_mode", "offline");
-    session_variables_.emplace("enable_trace", "false");
-    session_variables_.emplace("sync_job", "false");
-    session_variables_.emplace("job_timeout", "20000");  // ref TaskManagerClient::request_timeout_ms_
+    std::string db = openmldb::nameserver::INFORMATION_SCHEMA_DB;
+    std::string table = openmldb::nameserver::GLOBAL_VARIABLES;
+    std::string sql = "select * from " + table;
+    hybridse::sdk::Status status;
+    auto rs = ExecuteSQLParameterized(db, sql, std::shared_ptr<openmldb::sdk::SQLRequestRow>(), &status);
```
Comment (dl239) [inline src/sdk/sql_cluster_router.cc:265]: if (rs != nullptr)

File: src/cmd/sql_cmd_test.cc [hunk 3]
```diff
@@ -1474,6 +1474,7 @@ TEST_P(DBSDKTest, GlobalVariable) {
     auto cli = GetParam();
     cs = cli->cs;
     sr = cli->sr;
+    sr->Init();
```
Comment (aceforeverd) [inline src/cmd/sql_cmd_test.cc]: no need, did in `main` already: https://github.com/4paradigm/OpenMLDB/blob/d0c014210bf90d04b525202ff9c381c818d7f958/src/cmd/sql_cmd_test.cc#L1863

**Review by aceforeverd — APPROVED** [review]

File: src/cmd/sql_cmd_test.cc (hunk 3 above)
Comment (keyu813) [reply src/cmd/sql_cmd_test.cc]: but if we run all test, before this test, `session.execute_mode` will be set as `online`, so I init sr here to init session var and session var will as same as global var

**Review by dl239 — APPROVED** [review]

**Pull request merged**